{"name": "test_contact", "status": "broken", "statusDetails": {"message": "AttributeError: 'Contact' object has no attribute 'project_viability_xpath'", "trace": "self = <test_login.Test001Login object at 0x7f6aaeeb6e50>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"222d5bc64cb6bc7bef8839a6d93ecc05\")>\n\n    def test_contact(self,setup):\n>       co = Contact(setup)\n\ntestCases/test_login.py:258: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pageObjects.ProjectDetailsForm.Contact.Contact object at 0x7f6aaeeb6c10>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"222d5bc64cb6bc7bef8839a6d93ecc05\")>\n\n    def __init__(self,setup) -> None:\n         self.driver = setup\n>        link = utils.explicit_wait_present(self.driver,self.project_viability_xpath)\nE        AttributeError: 'Contact' object has no attribute 'project_viability_xpath'\n\npageObjects/ProjectDetailsForm/Contact.py:8: AttributeError"}, "start": 1629179904226, "stop": 1629179904226, "uuid": "67be22e0-2b05-4fe3-92b1-c267d0ee7d47", "historyId": "d2ef3188f225179002b53a9d65f2ce10", "testCaseId": "a088fd2763a36917aa78893f5f7d46fb", "fullName": "testCases.test_login.Test001Login#test_contact", "labels": [{"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test001Login"}, {"name": "host", "value": "bipin-Nitro-AN515-54"}, {"name": "thread", "value": "28037-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login"}]}